test/built-ins/Object/create/15.2.3.5-0-2.js

```diff
@@ -4,12 +4,6 @@
 /*---
 es5id: 15.2.3.5-0-2
 description: Object.create must exist as a function taking 2 parameters
-includes: [runTestCase.js]
 ---*/
 
-function testcase() {
-  if (Object.create.length === 2) {
-    return true;
-  }
- }
-runTestCase(testcase);
+assert.sameValue(Object.create.length, 2, 'Object.create.length: ');
```

Comment:
These assertion messages may work fine on failures, but maybe it's better to set it little bit more specific like `"Object.create.length === 2"` on any report uses.